steamos-fg

```diff
@@ -5,11 +5,14 @@ set -e
 exec "$@" & # start game
 sleep 5 # wait a bit for game window
 
+# find x11 display number
+DISPLAY_NUMBER=$(w -oush `whoami` | grep -Eo ' :[0-9]' | uniq | tr -d '[:space:]')
```

Comment:
This doesn't work for me on my Arch system, `DISPLAY_NUMBER` is blank. However, simply using the `$DISPLAY` environment variable seems to work fine for me.

Reply:
Interesting, I'd expect `who` to work everywhere :thinking: . `$DISPLAY` does not exist on my system - i' check against a fresh installation of ubuntu and see if it just has to do with my set up. Which ubuntu version would you like to target for (18.04 I guess)?

Reply:
Oh, sorry, that was a mistake, my ssh session does not have `$DISPLAY` but i can see in the processes running in the steamos session that their env indeed have `$DISPLAY` set. This got much more elegant all of a sudden :wink: i'll update the PR. Thanks for the feedback!

Reply:
Great, thanks for updating the PR.